Develop/scripts/script.js

```diff
@@ -0,0 +1,198 @@
+$(document).ready(function () {
+  var cityArray = [];
+  var lon = "";
+  var lat = "";
+  var apiKey = "39edc21fee70b725a0dbcb6dbed85366";
+  var city = "";
+
+  retrieveCityArray();
+
+  // When user searches for city
+  $("#add-city").on("click", function (event) {
+    event.preventDefault();
+
+    storeCityArray();
+    retrieveCityArray();
+  });
+
+  // When user clicks city button
+  $(document).on("click", ".city-button", getCityClicked);
```

Comment:
You probably want to add click handlers as you've done above. Any reason fo adding like this?